puppet/install_puppet_and_test_client.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 
+FILE="$1"
```

Comment:
Please do something like this to give an error if the parameter is not provided:
https://wiki.bash-hackers.org/syntax/pe#display_error_if_null_or_unset
`FILE=${1:?Specify puppet file path as command line parameter}`